lib/helpers/getUserReport.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var timeHelper = require('./time');
+
+module.exports = function getUserReport(standup) {
+  var color = '#000000'.replace(/0/g, function () {
+    return (~~(Math.random()*16)).toString(16);
+  });
+  var fields = [];
+  if (standup.yesterday) {
+    fields.push({
+        title: 'Yesterday',
+        value: standup.yesterday,
+        short: false
+    });
+  }
+  if (standup.today) {
+    fields.push({
+      title: 'Today',
+      value: standup.today,
+      short: false
+    });
+  }
+  if (standup.blockers) {
+    fields.push({
+      title: 'Blockers',
+      value: standup.blockers,
+      short: false
+    });
+  }
+  if (standup.goal) {
+    fields.push({
+      title: 'Goal',
+      value: standup.goal,
+      short: false
+    });
+  }
+
+  return {
+    title: timeHelper.getReportFormat(standup.date),
+    fields: fields,
+    color: color,
+    thumb_url: standup.thumbUrl
+  };
+};
```

Comment:
There are only two lines difference between this and `getStandupReport`, and they both apply to the title.  Instead of having another file that is basically identical, could we refactor how one of them works to accept the title as an argument and then use the same one in both places?  Or is it reasonable that user and standup reports may eventually be sufficiently different to justify handling them differently?

(You may already be addressing this in your refactor, in which case, ignore me.  😛)